packages/react/src/components/List/ListItem.tsx

```diff
@@ -9,27 +9,38 @@ import {
   ContentComponentProps,
   isFromKeyboard,
 } from '../../lib'
-import ItemLayout from '../ItemLayout/ItemLayout'
```

Comment:
👍 how difficult would be now to remove `ItemLayout` and `Layout` from our repo?

Reply:
I think the Label is the only other component that uses the Layout (I have to check again...) So no big blockers after this will get in..